xm-commons-domainevent-db/src/main/java/com/icthh/xm/commons/domainevent/db/service/DatabaseSourceInterceptor.java

```diff
@@ -0,0 +1,256 @@
+            return TABLE_NAME_CACHE.get(entity.getClass());
+        }
+
+        Table tableAnnotation = entity.getClass().getAnnotation(Table.class);
+
+        String tableName;
+        if (tableAnnotation != null) {
+            tableName = tableAnnotation.name();
+        } else {
+            log.warn(String.format("Entity: %s hasn't @Table annotation", entity.getClass().getSimpleName()));
+            tableName = findTableNameByHibernateNamingStrategy(entity);
+        }
+        TABLE_NAME_CACHE.put(entity.getClass(), tableName);
+
+        return tableName;
+    }
+
+    private String findTableNameByHibernateNamingStrategy(Object entity) {
+        Metamodel metamodel = entityManager.getMetamodel();
+
+        String tableName = "";
+        if(metamodel instanceof MetamodelImplementor metamodelImplementor) {
+            EntityPersister entityPersister = metamodelImplementor.locateEntityPersister(entity.getClass());
+            if (entityPersister instanceof AbstractEntityPersister abstractEntityPersister) {
+                tableName = abstractEntityPersister.getTableName();
+            }
+        }
+        log.info(String.format("Table name by hibernate name strategy: %s", tableName));
+        return tableName;
+    }
+
+    private boolean isIntercepted(String tableName, SourceConfig sourceConfig, JpaEntityContext context) {
+        if (isNull(sourceConfig.getFilter())) {
+            return false;
+        }
+
+        String key = sourceConfig.getFilter().getKey();
+        Boolean needFiltering = databaseFilter.lepFiltering(key, tableName, context);
+
+        if (needFiltering == null) {
+            // no lep executed, going to calculate DSL
+            Map<String, List<EntityFilter>> dsl = sourceConfig.getFilter().getDsl();
+
+            if (dsl.containsKey(tableName)) {
+                List<EntityFilter> filters = dsl.get(tableName);
+
+                Boolean invoked = lepInvoked(filters, tableName, context);
+
+                return Objects.requireNonNullElseGet(invoked, () -> anyMatchQuery(filters, context));
+            }
+            return false;
+        } else {
+            return needFiltering;
+        }
+    }
+
+    private Boolean lepInvoked(List<EntityFilter> filters, String tableName, JpaEntityContext context) {
+        Boolean invoked = null;
+
+        for (EntityFilter filter : filters) {
+            Boolean result = databaseDslFilter.lepFiltering(filter.getKey(), tableName, context);
+
+            if (TRUE.equals(result)) {
+                return true;
+            } else if (nonNull(result)) {
+                invoked = result;
+            }
+        }
+        return invoked;
+    }
+
+    private boolean anyMatchQuery(List<EntityFilter> filters, JpaEntityContext context) {
+        return filters.stream()
+            .anyMatch(filter -> processQuery(filter.getQuery(), context));
+    }
+
+    private boolean processQuery(Query query, JpaEntityContext context) {
```

Comment:
I propose to add logs to the trace level to query processing logic for simplify further debug